### PR TITLE
refactor: replace use of the deprecated ioutil package

### PR DESCRIPTION
**Pull request by EtienneM**
~Wait for https://github.com/Scalingo/go-plugins-helpers/pull/5 to be merged~

### CODE REVIEW

**Comment by TheoEb** [comment]
What's the issue here ? 
Should we fix the tests ? Sync with upstream ?

**Comment by EtienneM** [comment]
The specs are failing and I had no time to understand why. You should have a look at the specs and try to understand why they fail.

**Comment by TheoEb** [comment]
From what I understand, there is the library docker/docker that uses a `vendor.conf` file, and go mod don't care about it, so we have some dependencies that have not the right version.
I tried to disable go modules for this project (seems working) but it's not the best solution.

**Review by curzolapierre — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module github.com/Scalingo/go-plugins-helpers
 
-go 1.15
+go 1.16
 
 replace github.com/Sirupsen/logrus v1.7.0 => github.com/sirupsen/logrus v1.7.0
```
Comment (TheoEb) [inline go.mod:5]: Why is this needed ?

Comment (EtienneM) [reply go.mod:5]: This dependency got renamed from `Sirupsen` to `sirupsen` a while back and it caused a mess in many Go projects around the world.

But you're right, we may not need it in this project. Let me check that.

The comprehensive answer is here: https://github.com/sirupsen/logrus/issues/570#issuecomment-313933276

Comment (EtienneM) [reply go.mod]: I removed it in b60c82e
